js/adapt-assessmentQuestionBank.js

```diff
@@ -1,66 +1,51 @@
-import logging from 'core/js/logging';
+import data from 'core/js/data';
+import _ from 'underscore';
```

Comment:
do we need this import?

Reply:
No, but at some point we're going to have to put them all in.